macros/core/PGML.pl

```diff
@@ -1382,6 +1382,9 @@ sub Escape {
 	$string =~ s/</&lt;/g;
 	$string =~ s/>/&gt;/g;
 	$string =~ s/"/&quot;/g;
+
+	# Wrap the characters \, `, and $ in span tags to prevent MathJax from processing them.
+	$string =~ s/([\\`\$]+)/<span class="tex2jax_ignore">$1<\/span>/;
```

Comment:
You don't need the `class="tex2jax_ignore"`, here; just using a plain `span` will do.

Reply:
@dpvc Since I'm matching multiple characters at once, this could escape multiple backslashes, which I thought you mentioned would then be escaped by MathJax, `<span>\\\\\\</span>` for instance.  I also thought adding the class would help anyone who saw these spans in the html better understand their purpose.

Reply:
Sorry, didn't look carefully enough.  You are right.  I retract my comment.